### PR TITLE
Feature/caes 862 implement edit item api

### PR DESCRIPTION
**Pull request by cu4xxi**
Implement edit item api
Refactor Item component
Make trash view

### CODE REVIEW

File: packages/components/ItemFields/ItemHeader.js [hunk 1]
```diff
@@ -43,28 +58,44 @@ export const ItemHeader = ({ item }) => {
...
+            color="white"
+            onClick={onClickRemoveItem}
+          />
+        </>
+      ) : (
+        <>
+          <PathButton
+            color="white"
+            onClick={() => {
+              console.log('Change path');
```
Comment (beshkenadze) [inline packages/components/ItemFields/ItemHeader.js:78]: hm?

File: packages/components/ItemFields/ItemHeader.js [hunk 2]
```diff
@@ -43,28 +58,44 @@ export const ItemHeader = ({ item }) => {
...
+          >
+            Personal
+            <Delimeter>|</Delimeter>
+            Passwords
+          </PathButton>
+          <ActionButton
+            icon="share"
+            color="white"
+            onClick={() => {
+              console.log('Share');
```
Comment (beshkenadze) [inline packages/components/ItemFields/ItemHeader.js:89]: mm?

File: packages/components/ItemFields/ItemHeader.js (hunk 1 above)
Comment (cu4xxi) [reply packages/components/ItemFields/ItemHeader.js:78]: I implemented this functional in next pr.
So this console is absent now

File: packages/components/ItemFields/ItemHeader.js (hunk 2 above)
Comment (cu4xxi) [reply packages/components/ItemFields/ItemHeader.js:89]: The same

File: packages/components/ItemFields/ItemHeader.js (hunk 1 above)
Comment (beshkenadze) [reply packages/components/ItemFields/ItemHeader.js:78]: please use the notification (not-yet-implemented) instead of the console.log

**Pull request merged**